docs/editor_manual/browser_issues.rst

```diff
@@ -0,0 +1,11 @@
+Browser issues
+~~~~~~~~~~~~~~
+
+Some issues with Wagtail may come from the browser. Try changing or resetting its settings, or clearing the browser cache and cookies. You can also try using Wagtail with `another browser <http://browsehappy.com/>`_ to see if the problem persists.
```

Comment:
Since we're targeting editors in this manual. Could we provide them to a link on how-to clear their cache / cookies? For eg: http://www.refreshyourcache.com/en/home/